CMakeLists.txt

```diff
@@ -310,29 +351,16 @@ if (MAGMA_FOUND)
     "  MAGMA_LIBRARIES: ${MAGMA_LIBRARIES}"
   )
   include_directories(${MAGMA_INCLUDE_DIRS})
-else()
-  message(STATUS
-    "MAGMA not found"
-  )
-endif()
-
-if (MAGMA_FOUND)
   if(BML_CUSOLVER)
-    message(STATUS "Will use cuSOLVER")
     add_definitions(-DBML_USE_CUSOLVER)
   endif()
   if(BML_ROCSOLVER)
-    message(STATUS "Will use rocSOLVER")
```

Comment:
same here

Reply:
These STATUS messages were just moved to lines 306, 319, not deleted